main.rb

```diff
@@ -29,17 +30,61 @@ def choose_number(choice, app)
 end
 
 def main
-  app = App.new
+  # load the data and then parse it to the app
 
+  books = load_book_data
+  persons = load_person_data
+  rentals = load_rental_data(persons, books)
+  app = App.new(books, persons, rentals)
```

Comment:
- [ ] After solving the error I mentioned above, I added new data for books, persons, and rentals the saved it by exiting the app. When I tried to rerun the app after that I got the error in the image below. Please fix that.

![image](https://github.com/brighton0102/School-Library-App/assets/68796232/96191e0b-c56c-436f-9ee6-5642484fbc3e)